pims/display.py

```diff
@@ -481,11 +483,12 @@ def _to_rgb_uint8(image, autoscale):
     if ndim == 3 and shape.count(3) == 1:
         # This is a color image. Ensure that the color axis is axis 2.
         color_axis = shape.index(3)
-        image = np.rollaxis(image, color_axis, 3)
+        image = np.rollaxis(image, color_axis, 2)
```

Comment:
@nkeim I think this should remain 3. `rollaxis` rolls the axis until the designated one lies before the given number. We want to have the color axis at position `2`, so the last argument should be `3`.

    a = np.empty((3, 100, 100))
    print(np.rollaxis(a, a.shape.index(3), 2).shape)

Gives the incorrect result `(100, 3, 100)`.

Reply:
You're right! The first line of the `rollaxis()` docstring is a bit misleading…